packages/devtools_app/lib/src/shared/diagnostics/dart_object_node.dart

```diff
@@ -273,23 +274,31 @@ class DartObjectNode extends TreeNode<DartObjectNode> {
 
   @override
   bool get isExpandable {
+    final theRef = ref;
```

Comment:
I don't think we need to assign this to a new variable

Reply:
I need type promotion to access theRef.refNodeType little later

Reply:
Ah got it. Optional suggestion:

```
if (ref is ObjectReferences) {
  objectRef = ref as ObjectReferences;
  ... 
  return objectRef.refNodeType...
}